commands/init_test.go

```diff
@@ -1,32 +1,92 @@
 package commands
 
 import (
-	"github.com/github/hub/Godeps/_workspace/src/github.com/bmizerany/assert"
-	"github.com/github/hub/github"
+	"path/filepath"
+	"fmt"
 	"os"
-	"regexp"
 	"testing"
+
+	"github.com/github/hub/Godeps/_workspace/src/github.com/bmizerany/assert"
+	"github.com/github/hub/github"
 )
 
-func TestTransformInitArgs(t *testing.T) {
+func setup() {
```

Comment:
There's no setup/teardown for a test suite. Go 1.4 introduces `TestMain` (https://golang.org/doc/go1.4#minor_library_changes, scroll down the bullet point) but that's a global setup for all tests. However, there're other libraries support such feature, e.g., we switched to https://github.com/stretchr/testify in go-octokit which has setup/teardown for a test suite.

Regarding to the changes, how about renaming `setup()` to something less generic like `setupTestConfigs()`? The method is available to the whole `command` package and it'd be confusing for test cases in other files.